uiowa/uiowa
# Writing University blog entry: image migration module

Relates to #2792 





# How to test
Need a local copy of writinguniversity.org in Dev Desktop.
Pull the `vwu` multisite from the Graduate College Acquia Cloud docroot.
Leave the DD Apache/MySQL stack running.

`git checkout writinguniversity-migrate` and `git pull`
`blt ds --site=writinguniversity.org`
`drush @org-writinguniversity.local en writinguniversity_migrate`
`drush @org-writinguniversity.local mim d7_writinguniversity_blog`
Check http://org-writinguniversity.local.drupal.uiowa.edu/news and that blog entries from writinguniversity.org have been migrated in, that their teaser images/descriptions look good, and that the actual pages don't appear to have any issues (site-wide default for article image is set to do not display; featured image should only appear on the teaser)

Can also run `drush @org-writinguniversity.local mim d7_writinguniversity_blog_redirect`--sets up redirects from old url alias paths to the new url path. Might not be necessary to migrate.

# Note

This PR contains the module/config to run the migration for #2792, but the actual migration and pushing db/files up will need to be run separately, and will still be pending following merge.
